presidio-analyzer/tests/test_iban_recognizer.py

```diff
@@ -32,300 +32,860 @@ def update_iban_checksum(iban):
 @pytest.mark.parametrize(
     "iban, expected_len, expected_res",
     [
-        ("AL47212110090000000235698741", 1, ((0, 28),),),
-        ("AL47 2121 1009 0000 0002 3569 8741", 1, ((0, 34),),),
+        (
```

Comment:
Actually this is less readable as a parametrized scenario, will see how to exclude this :(